CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 ## dbt-teradata 1.0.0a
 
 ### Features
+* Merge incremental strategy is now supported in dbt-teradata
 
 ### Fixes
-
+* Fixed the bug (https://github.com/Teradata/dbt-teradata/issues/77)
 ### Docs
 
-### Under the hood
\ No newline at end of file
+### Under the 
```

Comment:
Is this change intentional?

Reply:
Sorry, that was by mistake, will rectify that.